Upload File: fixes

vikasranatalentelgia opened the pull request:
Create Folder Enable
Create folder validation(accept numbers)
Image view: if user add only one image and remove via a handle x button,
then upload file button is hide.
Image eclipse
Cursor pointer
searching date and name  on list view.
Delete functionality.
Open Folder is open on second click
On list View when click on folder name then open a folder.
Add File Button is hide ,when click on handle x button

vikasranatalentelgia committed:
Upload File: fixes

Create Folder Enable
Create folder validation(accept numbers)
Image view: if user add only one image and remove via a handle x button,
then upload file button is hide.
Image eclipse
Cursor pointer
searching date and name  on list view.
Delete functionality.
Open Folder is open on second click
On list View when click on folder name then open a folder.
Add File Button is hide ,when click on handle x button

vikasranatalentelgia committed:
Upload file: fix

Three Dots on Image View
Image Preview Details
folder searching Data

akuks commented:
@vikasranatalentelgia which issue is associated with this PR. Please mention in description

vikasranatalentelgia commented:
> @vikasranatalentelgia which issue is associated with this PR. Please mention in description

@akuks these issues came while doing unit testing, no tickets are generated yet for these

vikasranatalentelgia committed:
Merge branch 'development' into CAN-414

vikasranatalentelgia committed:
CAN-294,CAN-470, Upload file

Field layer is appearing twice. Error message when you clicked on back arrow , in login screen, upload file unit testing fixes
